src/common/simulation.cpp

```diff
@@ -92,10 +92,33 @@ void conquerspace::common::systems::simulation::Simulation::tick() {
         }
     }
 
-    BEGIN_TIMED_BLOCK(Orbit_Sim);
+    BEGIN_TIMED_BLOCK(Movement);
     auto bodies = m_universe.view<cqspt::Orbit>();
     for (entt::entity body : bodies) {
-        cqspt::updateOrbit(m_universe.get<cqspt::Orbit>(body));
+        auto &orb = cqspt::updateOrbit(m_universe.get<cqspt::Orbit>(body));
+        cqspt::updatePos(m_universe.get<cqspt::Kinematics>(body), orb);
     }
-    END_TIMED_BLOCK(Orbit_Sim);
+    auto bodies2 =
+        m_universe.view<cqspt::MoveTarget>(entt::exclude<cqspt::Orbit>);
+
+    for (entt::entity body : bodies2) {
+        cqspt::Kinematics & bodykin = m_universe.get<cqspt::Kinematics>(body);
+        cqspt::Kinematics & targetkin = m_universe.get<cqspt::Kinematics>(
+            m_universe.get<cqspt::MoveTarget>(body).targetent);
+        
+        SPDLOG_INFO(bodykin.postion.x);
```

Comment:
Have more informative logger information like tell what these variables, and put it into trace.